script.js

```diff
@@ -6,4 +6,75 @@ openMenu();
 function closeMenu() {
   document.getElementById('mobile-menu').style.display = 'none';
 }
-closeMenu();
\ No newline at end of file
+closeMenu();
+
+function popup() {
+  document.getElementById('pop-up-layer').style.display = 'flex';
+}
+popup();
+
+function closePopUp() {
+  document.getElementById('pop-up-layer').style.display = 'none';
+}
+
+closePopUp();
+
+const recentWorks = [
+  {
+    name: 'Multi-Post Stories Gain+Glory',
+    skillsDisplay: ['Ruby on rails', 'CSS', 'JavScript', 'html'],
+    submitButton: 'See Project',
+    upCardimg: '#!',
+  },
```

Comment:
- [ ] Kindly add all the necessary information required to generate your popup such as `image URL, description, live link`. 🧐️
You should generate the popup dynamically using the data stored here.